gradle/wrapper/gradle-wrapper.properties

```diff
@@ -2,5 +2,5 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.3.1-all.zip
-distributionSha256Sum=c5b67330a8a211539d713852c56a6a80fdea365d8902df92d1759d913d18fa2d
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.4-all.zip
+distributionSha256Sum=7a2c66d1a78f811d5f37d14630ad21cec5e77a2a4dc61e787e2257a6341016ce
```

Comment:
Btw, have you run `gradle wrapper` and there were no changes in the gradle-wrapper.jar/scripts or you just edited it manually?

Reply:
I don't remember. But updating to 4.4.1 via `gradle wrapper` results in a new jar file. So I guess I edited the properties file manually at the first time.